src/custom.d.ts

```diff
@@ -3,3 +3,6 @@ declare module '*.svg' {
   const content: CSSObject
   export default content
 }
+
+// @TODO: add declarations based on documentation.
```

Comment:
Toto plánuješ ještě vyřešit nebo mám vytvořit po merge separátní task? V druhém případě před merge komentář odstraním.

Reply:
Nee, neplanuju to zatim resit. V idealnim pripade by bylo fajn aby nekdo udelal PR na jejich repository. Ale je to dost casu s minimalni hodnotou pro nas tedka.